plugins/catalog-backend/src/ingestion/types.ts

```diff
@@ -14,8 +14,19 @@
  * limitations under the License.
  */
 
+import { Entity } from '@backstage/catalog-model';
+import { Location, LocationSpec } from '../catalog';
```

Comment:
Should we move `Location` to the `catalog-model` as well?

Reply:
Hm I guess, yeah! I am thinking that maybe I'll replace locations entirely with an entity kind instead at some point buuuuut that's a later discussion ;) Until then yes locations should probably be there